omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -273,9 +274,8 @@ protected PageableResult doSearch(RequestContext context) throws ResponseExcepti
 			else {
 				return new NeedsPaging<Order>(orders, context);
 			}
```

Comment:
Is this change related to the pull request?

Reply:
No, it got mixed up from another branch, let me see how I can fix that

Reply:
@dkayiwa is there any way I can drop the first commit (RESTWS-716) from this pull request?!